plugins/arDominionB5Plugin/js/fullWidthTreeView.js

```diff
@@ -163,18 +163,50 @@
       },
     };
 
-    // Declare listeners
-    // On ready: scroll to active node
-    var readyListener = function () {
-      var $activeNode = $('li[selected_on_load="true"]')[0];
+    function scrollToActive() {
+      var $activeNode;
 
+      $activeNode = $("li > a.jstree-clicked")[0];
       pager.updateMoreLink($moreButton, $resetButton);
 
       // Override default scrolling
       if ($activeNode !== undefined) {
-        $activeNode.scrollIntoView(true);
-        $("body")[0].scrollIntoView(true);
+        $activeNode.scrollIntoView(false);
       }
+    }
+
+    var showAlert = function (message, type) {
+      if (!type) {
+        type = "";
+      }
+
+      var $alert = $(
+        '<div class="alert ' +
+          type +
+          ' alert-dismissible fade show" role="alert">'
+      ).append(message);
+
+      var closeButton =
+        '<button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="' +
+        $(".index #fullwidth-treeview-present").data("treeview-alert-close") +
```

Comment:
Should this be `#fullwidth-treeview-capable` instead of `#fullwidth-treeview-present`?

Reply:
Good catch! Fixed.